<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTranslate</class>
 <widget class="QWidget" name="FileTranslate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileTranslate</string>
  </property>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>401</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReadPath">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>读取文件路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRead">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>读取并生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMakePath">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>选择路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>生成文件路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>401</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
